src/components/Shared/MessageHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import type { Profile } from '@generated/types';
+import getAvatar from '@lib/getAvatar';
+import type { FC } from 'react';
+import { useEffect } from 'react';
+import { useState } from 'react';
+
+import Follow from './Follow';
+
+interface Props {
+  profile?: Profile;
+}
+
+const MessageHeader: FC<Props> = ({ profile }) => {
+  const [following, setFollowing] = useState(true);
+
+  useEffect(() => {
+    setFollowing(profile?.isFollowedByMe ?? false);
+  }, [profile?.isFollowedByMe]);
+
+  console.log(following);
+
+  return (
+    <div className="flex justify-between flex-1 p-4 border-b-[1px]">
+      <div className="flex items-center">
+        <img
```

Comment:
Can we wrap this in a ```<Link href={`/u/${profile?.handle}`}>``` so we can click through to the person from the header?

Reply:
Done